apps/mobile/src/features/approver/components/approver-buttons.tsx

```diff
@@ -30,7 +30,7 @@ export function ApproverButtons({ approverState, onBack, onApprove }: ApproverBu
             buttonState="default"
             title={t({
               id: 'approver.button.approve',
-              message: 'Approve',
+              message: 'Confirm',
```

Comment:
Changing this to `Confirm` to match the other `message:`s for `'approver.button.approve'` and so that `lingui` can compile